### ISSUE TITLE
Upgrade Eddie documentation

### ISSUE THREAD

**Issue by ewallace** [riboviz/riboviz#230]
I'm addressing so using this issue to keep track of Eddie documentation issues I encounter.

* riboviz team users should use priority queue `bio_wallace_rna_riboviz`

I will address these in branch `eddie-docs-230`

**Comment by ewallace** [comment]
* suggest adding email address for job notification

**Comment by ewallace** [comment]
We also need more documentation on
* paths aka "where stuff goes" on Eddie, see [example-datasets#27](/riboviz/example-datasets/issues/27)
* unite this documentation with the job submission script? #228

Oh dear, a hairball of related issues.

**Comment by ewallace** [comment]
* suggest running with report, using nextflow option `-with-report report.html` ? Mentioned in #231.

**Comment by ewallace** [comment]
* use `-R y` flag to speed up job submission
* more on asking for fewer processes, with more memory
* even more memory may be needed if sample files (or genome) is very big
* reduce the number of processes in the yaml file(s) down to 4 by default

**Comment by ewallace** [comment]
Edits: I had best luck asking for entire nodes using a different parallel environment:
* Recommend `-pe mpi 16 -l h_vmem 4GB` to put on an entire node.

**Comment by kavousan** [comment]
@ewallace aims to close this together with #236.

**Comment by FlicAnderson** [comment]
Placeholder message to shame @FlicAnderson into editing this and adding my latest findings from testing Angela's latest incantation involving ringfencing (sent by email pre-Christmas). 
TL;DR: it runs ASAP, it's great!

**Comment by ewallace** [comment]
@FlicAnderson any change you could move that comment about ringfenced nodes to #236? I think it belongs there.

I've edited the documentation for `run-on-eddie.md` with things we know about now. Pull request #246 should allow us to close this issue.

**Comment by FlicAnderson** [comment]
@ewallace can we close this issue if it's merged and done?
